redirects.yaml

```diff
@@ -552,6 +552,7 @@ gateways/?:  "/internet-of-things/gateways"
 get/?: "/download"
 getubuntu/?: "/download"
 GetUbuntu/?: "/download"
+gov/?: "https:/canonical.com/public-sector"
```

Comment:
This is redirecting to `https://ubuntu-com-15271.demos.haus/canonical.com/public-sector` for me